iroh-resolver/src/resolver.rs

```diff
@@ -62,15 +62,25 @@ impl Path {
         &self.tail
     }
 
+    pub fn is_dir(&self) -> bool {
+        self.tail.len() > 0 && self.tail.last().unwrap().eq("")
```

Comment:
That's not guranteed to be true, that's at best and indication that this is a dir

Reply:
Technically it's intended use is purely string/browser path based manipulation, so it is a bit misleading, especially in the resolver crate. I don't se a sensible way to have info from the resolver if its a UnixFS::Dir before we get to the actual call we want to make when just parsing the path. 
The benefit is that this is only called/used when `FileResult::Directory` is returned by the client. I could potentially rename it or just separate out this logic into the handlers not to let others lean on it?

Reply:
maybe call it has_trailing_slash? then it is up to the caller to give it meaning